uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/DocumentTest.php

```diff
@@ -50,18 +50,6 @@ function testCreateDocumentWithNoSource() {
   /**
    * @expectedException CRM_Exception
    */
-  function testCreateDocumentWithNoAssignee() {
```

Comment:
I think it is better to keep these tests but change them a a positive tests , so ensure that even if you created a document with no assignee it will still go through and the document will be created ... same thing go for the tests below .